api/src/main/java/io/opentelemetry/metrics/LongValueRecorder.java

```diff
@@ -16,39 +16,54 @@
 
 package io.opentelemetry.metrics;
 
-import io.opentelemetry.metrics.LongMeasure.BoundLongMeasure;
+import io.opentelemetry.metrics.LongValueRecorder.BoundLongValueRecorder;
 import java.util.Map;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Measure to report instantaneous measurement of a long value.
+ * ValueRecorder is a non-additive synchronous instrument useful for recording any non-additive
```

Comment:
Same comment related to the use of "non-additive" in these docs.

Reply:
Done.